config.py

```diff
@@ -37,12 +37,13 @@ class BaseConfig(object):
     # Example:
     # MySQL: mysql+pymysql://{db_user}:{db_password}@{db_endpoint}/{db_name}
     # SQLite: sqlite:///local_data.db
-    DB_TYPE = os.getenv("DB_TYPE")
-    DB_USERNAME = os.getenv("DB_USERNAME")
-    DB_PASSWORD = os.getenv("DB_PASSWORD")
-    DB_ENDPOINT = os.getenv("DB_ENDPOINT")
-    DB_NAME = os.getenv("DB_NAME")
-
+    DB_TYPE = os.getenv("DB_TYPE", "")
+    DB_USERNAME = os.getenv("DB_USERNAME", "")
+    DB_PASSWORD = os.getenv("DB_PASSWORD", "")
+    DB_ENDPOINT = os.getenv("DB_ENDPOINT", "")
+    DB_NAME = os.getenv("DB_NAME", "")
+    DB_TEST_NAME = os.getenv("DB_TEST_NAME", "")
```

Comment:
Probably `os.getenv` takes None as default if not passed. If yes, then we don't need these changes

Reply:
I'll check

Reply:
> Probably `os.getenv` takes None as default if not passed. If yes, then we don't need these changes

It does but `{db_type}.format(db_type=None)` is `"None"` and not an empty string so that's why this change is required.